README.md

```diff
@@ -379,6 +380,32 @@ $ pip-audit --no-deps -r requirements.txt
 $ pip-audit --require-hashes -r requirements.txt
 ```
 
+## Tips and Tricks
+
+### Running against a `pipenv` project
+
+`pipenv` uses both a `Pipfile` and `Pipfile.lock` file to track and freeze dependencies
+instead of a `requirements.txt` file. `pip-audit` cannot process the `Pipfile[.lock]`
+files directly, however, these can be converted to a supported `requirements.txt` file
+that `pip-audit` can run against. Use a Python tool, such as
+[`pipfile-requirements`](https://github.com/frostming/pipfile-requirements), to
+convert your `Pipfile[.lock]` to a `requirements.txt` file and then run
+`pip-audit` against the generated requirements file.
+
+```console
+# install pipfile-requirements
+$ pipenv install --dev pipfile-requirements
+
+# Convert the Pipfile.lock to requirements.txt
+$ pipenv run pipfile2req Pipfile.lock > requirements.txt
+
+# Run pip-audit against the generated requirements.txt
```

Comment:
Some of the comments here are a little redundant; I'd suggest just collapsing it down to the commands themselves since they're relatively self-explanatory.

```suggestion
$ pipenv install --dev pipfile-requirements
$ pipenv run pipfile2req Pipfile.lock > requirements.txt
```